week-1-practise/level-2/1-tax.js

```diff
@@ -5,7 +5,9 @@
   Sales tax is 20% of the price of the product
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(price) {
+  return (20 * price) / 100;
+}
 
```

Comment:
There are many ways of achieving the same end goal here
```
return price * 0.2;

return price / 5;
```
You could even modify the parameters the function uses to take in a value for 'tax'. That way you could store a tax rate elsewhere (a database maybe) where it could be stroed once and used many time.